kagenti/examples/identity/auth_secret/README.md

```diff
@@ -0,0 +1,46 @@
+# `auth-secret`
+
+`auth-secret` is the image that creates a Keycloak client for Kagenti, gets the client secret, then creates a Kubernetes secret name `auth` that contains the client secret.
+
+# Kubernetes
+
+### Install Kagenti
+
+```sh
+cd kagenti/installer
+uv run kagenti-installer
+```
+
+### Change the `kagenti-ui` image
+
+Build the `ui-auth` image.
+
+```sh
+cd ui
+uv lock
+docker build -t ui-auth .
+```
+
+Inject the `ui-auth` image into kind cluster.
+
+```sh
+kind load docker-image ui-auth --name agent-platform
+```
+
+Change `kagenti-ui` deployment so it uses `ui-auth` image.
+
+```sh
+kubectl set image deployment/kagenti-ui \
+  kagenti-ui-container=ui-auth \
+  -n kagenti-system
```

Comment:
We should get the image from the same place the other images come from.  The user should not need to build and apply it on top of the previous no-auth image.